app/bg/background.js

```diff
@@ -127,6 +127,37 @@ let onCommandListener = function(cmd) {
 } //onCommandListener()
 chrome.commands.onCommand.addListener(onCommandListener);
 
+// When a window is created
+chrome.windows.onCreated.addListener(function(window){
+  // Leave TabFern window alone...
+  if (window.id === me.viewWindowID) return;
+  
+  if (S.getBool(S.FIXED_WINDOW_SIZE) && window.type === 'normal') {
+    let windowSize = S.getString(S.S_WINDOW_SIZE);
+    [windowWidth, windowHeight] = windowSize.split('x').map((value) => parseInt(value));
+    chrome.windows.update(window.id, {width: windowWidth, height: windowHeight});
+  }
+  if (S.getBool(S.PREFERRED_WINDOW_POSITION_OFFSET) && window.type === 'normal') {
+    let fixedOffsetFormat = RegExp(/^([0-9]+):([0-9]+)$/)
```

Comment:
nit: please change `let` to `const` here and elsewhere that we are creating regexes or other constants.

Also, same question about one field vs. two.